.github/workflows/wheels-manylinux-build.yml

```diff
@@ -208,8 +212,8 @@ jobs:
         uses-setup-env-vars: ${{ inputs.uses-setup-env-vars }}
 
         # secrets
-        sccache-aws-access-key-id: ${{ secrets.RAPIDSAI_GHA_AWS_ACCESS_KEY_ID }}
-        sccache-aws-secret-access-key: ${{ secrets.RAPIDSAI_GHA_AWS_SECRET_ACCESS_KEY }}
+        sccache-aws-access-key-id: ${{ env.AWS_ACCESS_KEY_ID }}
+        sccache-aws-secret-access-key: ${{ env.AWS_SECRET_ACCESS_KEY }}
```

Comment:
we probably also want to pass in the value of the new `AWS_SESSION_TOKEN` variable here.

Here are the AWS docs that talk about temporary credentials and the session token for reference.

- https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp_use-resources.html


CCing @sevagh for awareness about this change (no action is required).